compliance_checker/suite.py

```diff
@@ -315,6 +315,10 @@ def get_points(self, groups, limit):
 
         return score_list, points, out_of
 
+    # TODO: These methods for output generation could use a refactor to simply
```

Comment:
:+1:  to jinja templates
:-1: to `TODO` comments inside the code may get lost. Can you open an issue for that suggestion?